### PR TITLE
Fr 1593 consented gen letter judge

### PR DESCRIPTION
**Pull request by ocarleton**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FR-1593

### Change description ###

adding a new event for judge create general letter as its available states differ from court admin

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       "ccdUrl": "http://ccd-data-store-api-demo.service.core-compute-demo.internal"
     },
     "aat": {
-      "cosUrl": "http://finrem-cos-aat.service.core-compute-aat.internal",
+      "cosUrl": "http://finrem-cos-pr-376.service.core-compute-preview.internal",
```
Comment (ocarleton) [inline package.json:22]: to be removed before merge

File: definitions/contested/json/CaseEvent/CaseEvent.json [hunk 2]
```diff
@@ -693,5 +693,18 @@
...
     "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "FR_generalLetter_judge",
+    "Name": "Create General Letter",
+    "Description": "Create General Letter",
+    "DisplayOrder": 114,
+    "PreConditionState(s)": "consentOrderNotApproved;consentOrderApproved;consentOrderMade;consentProcess;caseWorkerReview;gateKeepingAndAllocation;schedulingAndHearing;prepareForHearing;caseFileSubmitted;judgeDraftOrder;reviewOrder;draftOrderNotApproved;solicitorDraftOrder;orderDrawn;scheduleRaiseDirectionsOrder;awaitingJudiciaryResponse",    "PostConditionState": "*",
```
Comment (shaunganley) [inline definitions/contested/json/CaseEvent/CaseEvent.json:704]: looks like formatting issue at the end

File: definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 3]
```diff
@@ -961,7 +961,7 @@
   {
     "LiveFrom": "01/06/2020",
     "CaseTypeID": "FinancialRemedyContested",
-    "CaseEventID": "FR_generalLetter",
```
Comment (shaunganley) [inline definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:964]: was this a bug or is this a case event that can now be removed?

File: definitions/contested/json/CaseEvent/CaseEvent.json (hunk 2 above)
Comment (ocarleton) [reply definitions/contested/json/CaseEvent/CaseEvent.json:704]: well spotted, fixed 👍

File: definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json (hunk 3 above)
Comment (ocarleton) [reply definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:964]: it was originally done adding the judge to this event, so he was granted access to the existing FR_generalLetter event.
However as the judge needs access on different states we needed to duplicate the event for the judge only. So this removes the judges access to the original event and gives him access instead to the judge specific one

**Pull request merged**